### PR TITLE
Remove the internal crypto algorithm enum that has to be maintained

### PR DESCRIPTION
**Pull request by devedup**
The algorithms are already downloaded from Apple, so just use this as the master list instead of keeping another internal lookup key list.

PHP isn't my main language, so if I've done something stupid..sorry..

I've tested this in my app, and it's working fine. Also ran the unit tests and updated the failing ones.

### CODE REVIEW

**Comment by hywak** [comment]
Thank you for pull request. I'm off till Monday. I will review the code on Tuesday afternoon or Wednesday

**Comment by hywak** [comment]
@Necsord maybe you will find a while to review the pull request earlier than me?

**Comment by devedup** [comment]
Cool. Certainly no rush for it on my side. Whenever you're ready 👍

File: src/Api/Factory/ResponseFactory.php [hunk 1]
```diff
@@ -56,15 +56,5 @@ private function validateAuthKey(array $authKey): void
                 )
             );
         }
-
```
Comment (hywak) [inline src/Api/Factory/ResponseFactory.php:59]: Please remove import form 5th line `use Azimo\Apple\Api\Enum\CryptographicAlgorithmEnum;`

File: src/Api/Response/JsonWebKeySetCollection.php [hunk 2]
```diff
@@ -19,16 +19,16 @@ public function __construct(array $authKeys)
 
     public function getByCryptographicAlgorithm(string $algorithm): ?JsonWebKeySet
     {
-        if (!CryptographicAlgorithmEnum::isSupported($algorithm)) {
+
```
Comment (hywak) [inline src/Api/Response/JsonWebKeySetCollection.php:22]: The only change is changed formatting.

```suggestion
    public function getByCryptographicAlgorithm(string $algorithm): ?JsonWebKeySet
    {
        $result = $this->authKeys[$algorithm] ?? null;
        if (!$result) {
            throw new UnsupportedCryptographicAlgorithmException(
                sprintf(
                    'Cryptographic algorithm `%s` is not supported.',
                    $algorithm
                )
            );
        }

        return $result;
    }
```

Comment (hywak) [inline src/Api/Response/JsonWebKeySetCollection.php:22]: Please remove import form 5th line use Azimo\Apple\Api\Enum\CryptographicAlgorithmEnum;

**Pull request merged**